content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,271 @@
+* The [Skupper](https://skupper.io/start/#step-1-install-the-skupper-command-line-tool-in-your-environment) CLI tool, the latest version 1.2.0. Use the [Skupper Installation](https://skupper.io/start/#step-1-install-the-skupper-command-line-tool-in-your-environment) to install this CLI tool to your environment. The _Skupper_ CLI tool works w﻿ith the same environment setup for _kubectl_ for accessing the public AWS EKS cluster and private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise. Some options, e.g., _--kubeconfig_, _--context_, and _--namespace_, can be used explicitly in _Skupper_ for using a specific _kubeconfig_ file and context, or accessing a Kubernetes namespace. 
+
+
+## Online Boutique
+
+[Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) is a cloud-first microservices demo application. It consists of an _11-tier_ microservices application. The application is a web-based e-commerce app where users can browse items, add them to the cart, and purchase them. This demo app has been used widely for demonstrating various technologies. It’s easy to deploy and it works on any Kubernetes cluster.
+
+This blog post will use the *Online Boutique* as our demo application, deploying it across the public AWS EKS cluster and the private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise using *Skupper*. 
+
+![](/img/apps.png)
+
+## Skupper
+
+[Skupper](https://skupper.io/) is a *Layer 7* service interconnect. It enables secure communication across multiple Kubernetes clusters through a Virtual Application Network (VAN). The VAN connects the applications and services in multiple clusters into a virtual network so that they can communicate with each other as if they were all running in the same site. VANs are able to provide connectivity across the hybrid cloud because they operate at Layer 7 (the application layer). They use Layer 7 application routers to route communication between Layer 7 application addresses. 
+
+With *Skupper*, your application can span multiple cloud providers, data centers, and regions with no VPNs or special firewall rules.
+
+## Deploy Online Boutique application
+
+Clone the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) GitHub repo to your local environment:
+
+```markdown
+$ git clone https://github.com/GoogleCloudPlatform/microservices-demo.git
+$ cd microservices-demo/release/
+```
+
+From the manifests file *kubernetes-manifests.yaml*  in the folder, create the following 3 manifests files:
+
+* **k8s-manifests-deploy-private.yaml**, including the following 3 *Deployment* manifests:
+
+  * *emailservice*;
```

Comment:
Are the semi-colons absolutely necessary after each bullet? For instance, is it a part of the name of each bullet descriptor? If not, I would not use the semi-colons. It makes it hard to read, especially in the italics font.